docs/products/add-data-to-grid-datastores/README.md

```diff
@@ -17,7 +17,33 @@ Datastores can be attached to Runs or Sessions and preserve the file format and
 We don't charge for data storage!
 :::
 
+## Datastores with Private S3 Buckets
 
+At this time we are only supporting usage of private S3 buckets as datastores for BYOC users. You can grant Grid access to your desired buckets by following the official aws [documentation](https://aws.amazon.com/premiumsupport/knowledge-center/cross-account-access-s3/).
+
+As a convenience, below we provided a bucket policy that grants Grid access to all the contents of your specified bucket. You can follow this official aws [documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/console_account-alias.html) to get your account id.
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::<aws-account-id-associated-with-byoc>:root"
```

Comment:
Hmm...technically here we can be more precise and directly specify the role for this cluster-id which will be doing datastore copying?

Reply:
and if you didn't specify one use root